docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* Less server interaction as in validating user input before sending page to server, thus, reducing server traffic.
+* Immediate feedback to the site visitors so that they dont reload page just to find they forgot to enter something.
```

Comment:
I tried to mention that User can see effects of their actions without requiring to reload like seeing any error messages on form validations etc.